### PR TITLE
[GOG-204] Override Plugins Temp Patch

### PR DESCRIPTION
**Pull request by ryanmrodriguez**
Temporary monkey patch for enabling failure-reasons plugin

### CODE REVIEW

File: app/assets/javascripts/plugins/failure-reasons.js [hunk 1]
```diff
@@ -0,0 +1,90 @@
+const API_URL = '/api/failure_reasons/',
```
Comment (benlangfeld) [inline app/assets/javascripts/plugins/failure-reasons.js:1]: Why put this in shipit?

File: app/models/shipit/deploy_spec.rb [hunk 2]
```diff
@@ -227,7 +227,7 @@ def review_checks
     end
 
     def plugins
-      config('plugins') || {}
+      config('plugins') || ["failure-reasons"]
```
Comment (benlangfeld) [inline app/models/shipit/deploy_spec.rb:230]: Why put this in shipit?

**Comment by benlangfeld** [comment]
This monkey patch should go in Milano, not shipit.

File: app/models/shipit/deploy_spec.rb (hunk 2 above)
Comment (ryanmrodriguez) [reply app/models/shipit/deploy_spec.rb:230]: @benlangfeld Just keeping you on your toes.... 👀

**Pull request closed without merging**